### ISSUE TITLE
Feature Request: Biomes not listed do not stop music

### ISSUE THREAD

**Issue by Duatao7055** [viniciusgf2/Ambience#22]
**Is your feature request related to a problem? Please describe.**
When I'm walking around my world, I come to river and beach biomes that I have no music specified for, but my current music is interrupted by silence whenever I enter an unspecified biome.

**Describe the solution you'd like**
When a biome is not specified in the Ambience config, that biome will continue playing music from the previous event/biome.

**Describe alternatives you've considered**
I deal with the music stops, create an area in the unspecified biome with the music that was playing in the previous biome, or just add those biomes to the list and deal with switching music for a few seconds.

**Additional context**

**Comment by viniciusgf2** [comment]
I didn't get it, do you want that when entering a biome that isn't specified that the music stops?

**Comment by Duatao7055** [comment]
If I didn't explain it properly, let me try again. For example, I have a river biome next to a forest biome. I want to cross the river, and I do not want the music to change from biome.Forest to biome.River. Right now if I have biome.Forest in the config but not biome.River in the config, this does not happen. What happens is I leave the forest biome, enter the river biome, and the music completely stops. The log is spammed with an error basically saying it could not find the music entry for that biome, and the music will start from the beginning again when I enter the Forest biome again. A solution for the log spamming and the music situation would be to make it so an unspecified biome event is completely ignored, and music from the previous event continues playing. An example of this solution would be that I don't have biome.River in the config and do have biome.Forest in the config, and in game that means that when I leave the Forest biome, or any biome, and enter a River biome, the music from the previous biome plays.

**Comment by viniciusgf2** [comment]
oh this should not happen, the default behavior is to continue playing the last found music, this kind of spamming error only happens if you have a event.river specified in the ambience.properties file but you don't have a river.mp3 (or something.mp3) in the musics folder

**Comment by Duatao7055** [comment]
Let me check again then

**Comment by viniciusgf2** [comment]
hm... this could be a bug this error this error is called by the java file manager when trying to loading the .mp3 and should only happen when it doesn't find the file, I will do some tests tomorow

**Comment by TheComputerizer** [comment]
I am having something similar, but not necessarily for biomes. Sometimes when I fall from above y 128, the music gets interrupted and starts a different song from what I have listed (generic). This also happens when I have not hit a mob for a while and then I do. Brief silence occurs and then the next song start playing (dimension). I do not have songs set for either high up or battle.

**Comment by viniciusgf2** [comment]
> I am having something similar, but not necessarily for biomes. Sometimes when I fall from above y 128, the music gets interrupted and starts a different song from what I have listed (generic). This also happens when I have not hit a mob for a while and then I do. Brief silence occurs and then the next song start playing (dimension). I do not have songs set for either high up or battle.

When you fall for more than 15 blocks it plays a falling song that is hardcodded and the battle song is supposed to always have a song configured, I've never tested to play without the battle song so strange things can happen

**Comment by TheComputerizer** [comment]
Yeah I hit a mob and it just skips to the next song out of nowhere. Usually it happens in the middle of a song too and makes for a pretty weird and unintended transition

**Comment by TheComputerizer** [comment]
https://streamable.com/n217tq Here is an example of what I am talking about. I reload, it starts playing the music, then I hit a mob and it just skips to the next song. I do not have a battle event in my properties.
